pyproject.toml

```diff
@@ -0,0 +1,96 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "array-api-compat"
+dynamic = ["version"]
+description = "A wrapper around NumPy and other array libraries to make them compatible with the Array API standard"
+readme = "README.md"
+requires-python = ">=3.9"
+license = {file = "LICENSE"}
+authors = [{name = "Consortium for Python Data API Standards"}]
+classifiers = [
+    "License :: OSI Approved :: MIT License",
```

Comment:
I had to upper cap setuptools both here and in `array-api-strict` : newer versions complained about this way of specifying the license being not SPDX-compliant (whatever that is).

Reply:
I mean earlier today (ok, yesterday, March 20), both here (setup.py) and in array-api-compat (pyproject.toml). The upper cap is a band-aid, so I was hoping you know about this SPDX thing. If not, then not, we'll upper cap for the time being.

And BTW, I'm planning to take a closer look at this PR, just not this week already.

Reply:
> And BTW, I'm planning to take a closer look at this PR, just not this week already.

No rush, and thank you!

>  I was hoping you know about this SPDX thing.

I'll look into it.  Do you know which command gave you the error?  I use `uv pip` now, so I didn't see any error.  (It did give me an error when I had both the license text and file though.)

Reply:
> Do you know which command gave you the error?

This was on CI, so `$ python -m build`, I'd think. Not sure if this matters, as long as setuptools is uber-recent

Reply:
Perfect, thanks!  Should be fixed now.  I removed the LICENSE file since it's supplanted by whatever you put into that field.  If you want to support a particular version of MIT  (or versions) of MIT, please let me know.  The spec is [here](https://spdx.github.io/spdx-spec/v2-draft/SPDX-license-expressions/#d1-overview).